README.md

```diff
@@ -1 +1,102 @@
-# open_vocabularies
\ No newline at end of file
+# Open Vocabularies
+
+The Open Vocabularies component allows you to create “open” vocabularies. These vocabularies wrap categorisation systems, like Drupal taxonomies or Publication Office vocabularies, or subsets of them.
+It provides a field type that can be added to fieldable entities. Then associations between the vocabularies and the field instances can be created, allowing to categorise the content in a flexible way.
```

Comment:
Shall we try with something like this?

The Open Vocabularies module allows users to choose how a piece of content will be categorised without having to change the related content type field definition.
It does that by providing:

- An open vocabulary field type
- The possibility to create associations between open vocabulary field types and Drupal categorisation systems, such as Drupal taxonomies, Publication Office vocabularies, or subsets of them.

In other words: once a content type is equipped with an open vocabulary field, users with appropriate access can decide which type of entities that field will able to referencing, by creating associations.